src/scale/Ordinal.ts

```diff
@@ -99,6 +103,23 @@ class OrdinalScale extends Scale {
         return;
     }
 
+    setCategorySortInfo(info: OrdinalSortInfo[]): void {
+        this._categorySortInfo = info;
+    }
+
```

Comment:
I can't find the general logic of sorting the categories when `realtimeSort` is not set. Did I miss it?

Reply:
It's not implemented yet. It will be fixed later.